reana_workflow_controller/rest/workflows.py

```diff
@@ -237,7 +247,19 @@ def get_workflows(paginate=None):  # noqa
                 "status": workflow.status.name,
                 "user": user_uuid,
                 "created": workflow.created.strftime(WORKFLOW_TIME_FORMAT),
-                "progress": get_workflow_progress(workflow),
+                "run_started_at": (
+                    workflow.run_started_at.strftime(WORKFLOW_TIME_FORMAT)
+                    if workflow.run_started_at
+                    else None
+                ),
+                "run_finished_at": (
+                    workflow.run_finished_at.strftime(WORKFLOW_TIME_FORMAT)
+                    if workflow.run_finished_at
+                    else None
+                ),
+                "progress": (
+                    get_workflow_progress(workflow) if include_progress else {}
```

Comment:
If `include_progress` is True, would it mean that `run_started_at` and `run_finished_at` information is repeated at two different levels?

Reply:
I guess `verbose` should be checked here as well to keep backward compatibility?

Reply:
Fixed the repetition.